verl/trainer/config/ppo_trainer.yaml

```diff
@@ -31,6 +33,9 @@ actor_rollout_ref:
     ppo_max_token_len_per_gpu: 16384 # n * ${data.max_prompt_length} + ${data.max_response_length}
     grad_clip: 1.0
     clip_ratio: 0.2
+    clip_ratio_low: 0.2
+    clip_ratio_high: 0.2
+    use_token_level_loss: False
```

Comment:
It appears that the behavior for computing pg loss prior to add the `use_token_level_loss` flag was already at token level. Should we set this to `True` for sake of backward compatibility?